### DISCUSSION TITLE
When will version 0.5 be released?

### DISCUSSION THREAD

**Discussion by ghost** [florisboard/florisboard#2506] [Q&A] [answered]
The app is very easy to use and I like it a lot, but it doesn't have the language I use right now. I'm looking forward to seeing if there are any new updates. Do you have a specific update time? I know it's complicated to develop an app, but I'd love to use a new version of the app

**Comment by lm41** [comment, marked as answer]
The next major release will be 0.4.0. Since all the people working on this project are working on it voluntarily, no ETAs will be released, so that nobody is put under pressure. Now that that's said, I can say that it won't be long before said version is released. (It may still take up to two months)

**Reply by ghost** [reply]
Thank you for your answer.  Wish your a happy life
